gateway-ha/src/test/java/io/trino/gateway/ha/handler/TestQueryIdCachingProxyHandler.java

```diff
@@ -66,6 +75,186 @@ public void testExtractQueryIdFromUrl()
                 .isNull();
     }
 
+    @Test
+    void testQueryIdFromKill()
```

Comment:
What's the expected behavior when the 1st argument isn't a single string literal? 
```
CALL kill_query('20200416' || '_160256_03078_6b4yt')
```

Reply:
IMO these should be rejected, since in general Trino Gateway cannot resolve arbitrary expressions. I'll add an explicit check